possel/application.py

```diff
@@ -110,26 +112,31 @@ def get_arg_parser():
 def main():
     args = get_arg_parser().parse_args()
```

Comment:
Possibly, either way add a "default_configuration" dict somewhere near the args and use that.